README.md

```diff
@@ -7,17 +7,29 @@
 
 ## Overview
 
-This package defines a [gRPC](https://grpc.io/) client library for Common Lisp.
-It wraps gRPC core functions with CFFI calls and uses those core functions to
-create and use a client. client.lisp contains all the necessary functions to
+This package defines a [gRPC](https://grpc.io/) library for Common Lisp.
+It wraps gRPC core functions with CFFI calls.
+
+``client.lisp`` contains all the necessary functions to
 create a gRPC client by creating channels (connections between client and
 server) and calls (requests to a server).
-
 Currently there is support for synchronous and streaming calls over
-
 SSL, and insecure channels.
 
-Support for implementing gRPC servers is in development.
+``server.lisp`` contains a ``run-grpc-server`` function that implements an insecure gRPC server.
```

Comment:
Let's stick with a single backquote everywhere unless double backquote is needed for escaping.